src/components/shared/ManagedTree.js

```diff
@@ -49,18 +49,13 @@ class ManagedTree extends Component<Props, State> {
 
   componentWillReceiveProps(nextProps: Props) {
     const { listItems, highlightItems, focused } = this.props;
-    if (
-      nextProps.listItems &&
-      nextProps.listItems != listItems &&
-      nextProps.listItems.length
-    ) {
+    if (nextProps.listItems && nextProps.listItems != listItems) {
       this.expandListItems(nextProps.listItems);
     }
 
     if (
       nextProps.highlightItems &&
-      nextProps.highlightItems != highlightItems &&
-      nextProps.highlightItems.length
+      nextProps.highlightItems != highlightItems
```

Comment:
last time we needed length because we were checking that the array had more than 0 items, is that no longer needed?